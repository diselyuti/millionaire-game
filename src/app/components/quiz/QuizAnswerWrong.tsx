import React from 'react';
import styles from '@/app/styles/components/quiz/quiz-answer.module.scss';

function QuizAnswerWrong({ letter, text }: { letter: string; text: string }) {
  return (
    <div className={`${styles.hexagon} ${styles.wrong}`}>
      <p className={styles.answerText}>
        <span className={styles.letter}>{letter}</span>
        <span>{text}</span>
      </p>
    </div>
  );
}

export default QuizAnswerWrong;
