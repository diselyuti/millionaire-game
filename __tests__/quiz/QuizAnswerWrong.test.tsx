import React from 'react';
import '@testing-library/jest-dom';
import QuizAnswerWrong from "@/app/components/quiz/QuizAnswerWrong.tsx";
import {render} from "@testing-library/react";

describe('QuizAnswerWrong', () => {
  test('should render texts', () => {
    const { container } = render(<QuizAnswerWrong letter="A" text="Answer" />);
    expect(container).toHaveTextContent('A');
    expect(container).toHaveTextContent('Answer');
  });

  test('should render QuizAnswerWrong', () => {
    const { container } = render(<QuizAnswerWrong letter="A" text="Answer" />);
    expect(container).toMatchSnapshot();
  });
});
