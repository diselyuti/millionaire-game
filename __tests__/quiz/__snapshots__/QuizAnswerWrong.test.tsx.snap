// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizAnswerWrong should render QuizAnswerWrong 1`] = `
<div>
  <div
    class="hexagon wrong"
  >
    <p
      class="answerText"
    >
      <span
        class="letter"
      >
        A
      </span>
      <span>
        Answer
      </span>
    </p>
  </div>
</div>
`;
